indexer-app/test/cmr/indexer/test/data/concepts/keyword_util.clj

```diff
@@ -319,87 +319,87 @@
                                              :MiddleName "D."
                                              :LastName "Eve"}]}]})
 
-(deftest fields->fn-mapper-single-valued
+(deftest extract-field-single-valued
   (is (= "An abstract summary"
```

Comment:
It seems like it might be a good idea to replace all these `is` calls with `are2`.

Reply:
Yeah, that is a good idea.